cfme/tests/infrastructure/test_providers.py

```diff
@@ -58,7 +58,7 @@ def test_add_cancelled_validation():
 def test_type_required_validation():
     """Test to validate type while adding a provider"""
     prov = InfraProvider()
-    with error.expected(FlashMessageException):
```

Comment:
@psav, this was @mfalesni request. I can change it back to old patter if you wish.